james-controller-consul/src/main/java/com/tomtom/james/controller/consul/ConsulController.java

```diff
@@ -149,7 +149,9 @@ private void onInformationPointsCleared(InformationPointService informationPoint
     protected Optional<InformationPoint> eventToInformationPoint(final ConfigurationEvent event) throws IOException {
         String methodReference = readMethodReference(event);
         String informationPointDtoAsJsonString = (String) event.getPropertyValue();
+        informationPointDtoAsJsonString = informationPointDtoAsJsonString.replace("\\\"","'");
```

Comment:
it looks very risky what if we have code like this `println \\"'\\"`
this code will change it in `println '''` which is probably invalid groovy